selenium.go

```diff
@@ -330,6 +362,12 @@ type WebDriver interface {
 	// ButtonUp causes the left mouse button to be released.
 	ButtonUp() error
 
+	// PerformActions executes stored actions.
```

Comment:
// PerformActions executes actions previously stored by calls to StorePointerActions and StoreKeyActions.

Reply:
done